client/landing/gutenboarding/hooks/use-on-site-creation.ts

```diff
@@ -78,6 +79,7 @@ export default function useOnSiteCreation(): void {
 	const flowCompleteTrackingParams = {
 		isNewSite: !! newSite,
 		isNewUser: !! newUser,
+		isBlankCanvas: design && isBlankCanvasDesign( design ),
```

Comment:
Since `isBlankCanvasDesign` accepts undefined.

```suggestion
		isBlankCanvas: isBlankCanvasDesign( design ),
```

Reply:
Thanks Omar! You're absolutely right and I committed your suggestion.